results/OAEI2025/results.md

```
@@ -29,6 +29,7 @@ The raw results can be found in the `raw-results_archtrack_2025` folder in this
 - Running without code errors / exceptions, alignments empty
     - LSMatch
     - LSMatch-Multilingual
+    - LogMap lite
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)